core/src/rpc.rs

```diff
@@ -1693,7 +1693,7 @@ fn deserialize_bs58_transaction(bs58_transaction: String) -> Result<(Vec<u8>, Tr
     }
     bincode::config()
         .limit(PACKET_DATA_SIZE as u64)
-        .deserialize(&wire_transaction)
+        .deserialize_from(&wire_transaction[..])
```

Comment:
I'm tempted to make this a plain-old `bincode::deserialize`, since we check the wire_transaction length and return a better error message immediately before this.

Reply:
If there isn't a performance impact I would say keep this consistent by using `deserialize_from`.  Keeping it is also a good measure in case the other checks change or are removed for some reason.

Reply:
Sounds good. That's where I landed also